spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubProperties.java

```diff
@@ -32,10 +32,14 @@
 @ConfigurationProperties("spring.cloud.gcp.pubsub")
 public class GcpPubSubProperties implements CredentialsSupplier {
 
-	// Contains settings specific to the subscriber factory
+	/**
+	 * Contains settings specific to the subscriber factory
```

Comment:
I know, I did a lot of spam in this PR, but let me one more time to point to the rule about this kind of JavaDocs: https://docs.spring.io/spring-boot/docs/2.0.2.RELEASE/reference/htmlsingle/#configuration-metadata-property-attributes

> A short description of the group that can be displayed to users. If no description is available, it may be omitted. It is recommended that descriptions be short paragraphs, with the first line providing a concise summary. The last line in the description should end with a period (`.`).

The main point is a `period` in the end of the sentence.

Would you mind to fix all of them in this class?

Thanks

Reply:
Alright, will add the period. (Just want to note that in English we do not add periods to the end of incomplete sentences.)

Reply:
Thanks for the English lesson :smile: ! That's English, but here it is JavaDocs, more over a description for the auto-configuration property and there is a rule in Spring Boot. That's what I request - consistency and clean Spring code. Fair?